server/grpc_service.go

```diff
@@ -218,6 +218,7 @@ func (s *Server) PutStore(ctx context.Context, request *pdpb.PutStoreRequest) (*
 	}
 
 	log.Infof("put store ok - %v", store)
+	cluster.cachedCluster.OnChangeClusterVersion()
```

Comment:
If the machine crashes before here will the cluster version not get updated?

Reply:
You are right. I need to handle this situation.

Reply:
@gregwebs I reconsider it. The cluster version will not get updated crashes before here but also the corresponding TiKV will not start up. it will register in next time when restarting TiKV and get the update.